.github/ISSUE_TEMPLATE/release-plan.md

```diff
@@ -30,10 +32,21 @@ assignees: 'Guite, craigh'
 
 - [ ] Start release process in core manager at <https://ziku.la>
   - [ ] Create QA ticket
-  - [ ] Create release
-  - [ ] Copy assets (note: it now downloads assets from the chosen GitHub Actions build, since Jenkins has been removed)
+  - [ ] Create tag for core project
+  - [ ] Create tag for all slave repositories
+- [ ] Wait a bit for packagist to update
```

Comment:
this wait may also need to wait for subtree split to complete which could take 12+ hours.